README.md

```diff
@@ -233,11 +233,32 @@ wrapper as follows:
 bluealsa-aplay -L
 ```
 
+## Release Policy
+
+The bluez-alsa project does not create nor distribute release software
+packages. However, at irregular intervals, when an important new feature is
+considered stable then the `master` branch is tagged with a release version
+number. These tags are included only to mark reference baselines; their main
+purpose is to help Linux Distribution package maintainers to relate their own
+package versioning schemes to the original codebase.
+
+A release tag is formed as `vN.N.N` where the first number indicates a change
```

Comment:
If possible I would add a link to https://semver.org/ somewhere.

Reply:
done